ansible/roles/operator-pipeline/templates/openshift/tasks/commit-pinned-digest.yml

```diff
@@ -23,6 +23,9 @@ spec:
         Binding a Secret to this Workspace is strongly recommended over other
         volume types.
   params:
+    - name: gitInitImage
```

Comment:
All other params use snake case formatting, but this one uses camel case. Can you please alight this param with others please?

Reply:
Oh... I see it is already merged... nvm

Reply:
There was `pipeline_image` already so i followed this. I think that `gitInitImage` is convention from `git-clone.yml` and i am guessing that this file was copied from `https://github.com/tektoncd/catalog/blob/main/task/git-clone/0.1/git-clone.yaml` so i left conventions from there